### ISSUE TITLE
【SERIOUS BUG】Learning Rate does not Change when use LION Optimizer

### ISSUE THREAD

**Issue by Rika-Mipa** [bmaltais/kohya_ss#248]
My friends and I found that if we use LION Optimizer to train model, then the learning rate can not be found in Tensorboard.
The learning rate diagram can be correctly found in Tensorboard when we use DAdaptation Optimizer.
Besides that, the features of charactor(such as hair and skirt) will showed in background, no matter how we change prompts or resolution,
It seems that there are some serious bugs with LION Optimizer(we think the problem is: currently, if we use LION, the lr will not change correctly)
